CSS: Remove padding for buttons in header actions list

mjansenDatabay opened the pull request:
See: https://mantis.ilias.de/view.php?id=45529

This PR fixes an obsolete `padding-left` for buttons in the header actions list.

If approved, please pick this to all relevant branches.

catenglaender commented:
Apologies for the late reply. :sweat_smile:  Thank you very much for the contribution! :tada: 
Could you please investigate if the whole styling of `div.ilHeadAction ul.dropdown-menu button` is needed at all? I am relatively confident that the whole thing was a band aid that is no longer required, especially if the dropdown has already been migrated to the Kitchen Sink dropdown. It's a small thing, but every line of code we can get rid of is a step towards lighter and more clean style code.
Very much appreciated! :pray:

catenglaender commented on templates/default/070-components/legacy/_component_headline.scss:
```diff
@@ -64,7 +64,6 @@ div.ilHeadAction a:hover {
 div.ilHeadAction ul.dropdown-menu button {
```
This entire style rule might not be needed if this is now a kitchen sink dropdown.

mjansenDatabay replied:
Thanks for the review @catenglaender, I updated the PR.

Please note that the dropdown is a legacy `ilAdvancedSelectionListGUI`, **not** a KS dropdown. Nevertheless, removing `div.ilHeadAction ul.dropdown-menu button` seems to not have any negative effect, so I think we could completely remove it.

<img width="934" height="177" alt="Bildschirmfoto vom 2025-09-18 09-26-00" src="https://github.com/user-attachments/assets/7d3545ac-4616-4699-99db-ee5a9d8906a3" />

mjansenDatabay committed:
CSS: Remove padding for buttons in header actions list

See: https://mantis.ilias.de/view.php?id=45529

This PR fixes an obsolete padding-left for buttons in the header actions list.

If approved, please pick this to all relevant branches.

catenglaender commented:
Thank you for looking into this. Yeah legacy and KS dropdowns share most of their code CSS style which is sometimes great, sometimes not so much. But in this case the modern code seems to be covering what used to be a patch. Very nice!

Thanks for the update! Very much appreciated!

mjansenDatabay commented:
@catenglaender Do you need additional PRs for ILIAS >= 10.x, or will you pick the commit yourself?

catenglaender commented:
I will pick them now :)
